vanilla-framework/vanilla-framework
# Add namespacing to global functions

## Done

Add namespacing to global functions

## QA

- Check this PR passes Travis testing
- Check this PR passes Percy testing

## Details

Fixes #1321
